src/components/sbb-journey-header/sbb-journey-header.custom.d.ts

```diff
@@ -1,5 +1,4 @@
 export interface InterfaceJourneyHeaderAttributes {
-  appearance: 'primary' | 'primary-negative';
-  markup: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'p' | 'span';
-  size: '4' | '5';
+  level: '1' | '2' | '3' | '4' | '5' | '6';
```

Comment:
please remove here and in tsx file import directly from title `InterfaceTitleAttributes['level']`
```suggestion
```